wagtailflags/views.py

```diff
@@ -43,6 +44,27 @@ def create_flag(request):
     return render(request, "wagtailflags/flags/create_flag.html", context)
 
 
+def delete_flag(request, name):
+    """ Delete a database flag. """
+    flag = get_flags().get(name)
+
+    if not flag:
+        raise Http404
+
+    if not deletable(flag):
+        return HttpResponseForbidden(request.method)
```

Comment:
```suggestion
        return HttpResponseForbidden(flag)
```

Reply:
Ack, I think I suggested the wrong thing here. This should be the `name` of the flag, not the `flag` object itself.